addons/calendar/static/src/js/base_calendar.js

```diff
@@ -33,20 +33,52 @@ const Many2ManyAttendee = FieldMany2ManyTagsAvatar.extend({
     // as this widget is model dependant (rpc on res.partner), use it in another
     // context probably won't work
     // supportedFieldTypes: ['many2many'],
-    tag_template: "Many2ManyAttendeeTag",
     specialData: "_fetchSpecialAttendeeStatus",
+    className: 'o_field_many2manytags avatar',
+
+    init: function () {
+        this._super.apply(this, arguments);
+        this.className += this.nodeOptions.block ? ' d-block' : '';
+    },
 
     //--------------------------------------------------------------------------
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * @override
+     * @private
+     */
+    _renderTags: function () {
+        this._super.apply(this, arguments);
+        const avatars = this.el.querySelectorAll('.o_m2m_avatar');
+        for (let idx = 0; idx < avatars.length; idx++) {
+            const partner_id = parseInt(avatars[idx].dataset["id"]);
+            const partner_data = this.record.specialData.partner_ids.filter(partner => {return partner.id === partner_id});
```

Comment:
Since we just use the first `partner_data` below, using `find` instead of `filter` would make more sense here, don't you think?

```suggestion
            const partner_data = this.record.specialData.partner_ids.find(partner => partner.id === partner_id);
```